compiler/sigmatch.nim

```diff
@@ -133,14 +133,6 @@ proc initCandidate*(ctx: PContext, c: var TCandidate, callee: PType) =
   initIdTable(c.bindings)
 
 proc put(c: var TCandidate, key, val: PType) {.inline.} =
-  when false:
```

Comment:
useful to keep for debugging (I've actually used it) ; maybe add a comment instead of deleting:
```nim
when false: # debugging
   ...
```